File: lib/Location.dart
class Location {
  final double latitud;
  final double longitud;

  const Location({required this.latitud, required this.longitud});

  double getLatitud() {
    return latitud;
  }

  double getLongitud() {
    return longitud;
  }

  factory Location.fromJson(Map<String, dynamic> json) =>
      Location(latitud: json['latitud'], longitud: json['longitud']);
}

File: API_FLUTTER/ApiFlutter/ApiFlutter/Models/Location.cs
using Microsoft.EntityFrameworkCore;
using System.ComponentModel.DataAnnotations;

namespace ApiFlutter.Models
{
    [Keyless]
    public class Location
    {
        [Required]
        public double latitud { get; set; }
        [Required]
        public double longitud { get; set; }


    }
}
